enterprise_access/apps/api/serializers.py

```diff
@@ -207,67 +192,136 @@ def validate(self, attrs):
         return attrs
 
 
-class SubsidyAccessPolicyRedeemListSerializer(SubsidyAccessPolicyRedeemSerializer):  # pylint: disable=abstract-method
+class ValidateContentKeyMixin:
     """
-    Serializer to validate policy request GET query params.
+    Mixin to provide validation function for the `content_key` serializer field.
+
+    Requires the inheriting serializer to declear a `content_key` field.  Supports one or many content keys.
     """
-    enterprise_customer_uuid = serializers.UUIDField(required=True)
+    def validate_content_key(self, value):
+        """
+        Validate `content_key` field.
+        """
+        content_keys = [value] if isinstance(value, str) else value
+        for content_key in content_keys:
+            try:
+                CourseKey.from_string(content_key)
+            except InvalidKeyError as exc:
+                raise serializers.ValidationError(f"Invalid content_key: {content_key}") from exc
+        return content_key
```

Comment:
[clarification] Is this intended to be `return content_keys`?

Reply:
supposed to return `value` actually, since it's designed to work with either a ListField or a StringField